ui/src/Routes.tsx

```diff
@@ -18,6 +19,15 @@ export const Router: React.FC = () => {
             </ProtectedRoute>
           }
         />
+
+        <Route
+          path="/clubs/:clubName"
```

Comment:
nit: I don't like `clubName` to be in the url. Maybe we need a `slug` field in the club model, which will be a part of UI url.